src_features/performPrivacyOperation/feature_performPrivacyOperation.h

```diff
@@ -1,3 +1,8 @@
+#ifndef _PERFORM_PRIVACY_OPERATION_H_
```

Comment:
I prefer the `#pragma once` pattern but as you want

Reply:
Just to be consistent with other headers in `src_features/*`...